extensions/amp-list/validator-amp-list.protoascii

```diff
@@ -213,6 +213,7 @@ tags: {  # <amp-list>
     value: "always"
     value: "no"
     value: "refresh"
+    value: "refresh-evaluate" # Experimental, email-only.
```

Comment:
@twifkak Small validator change review please? :)

Reply:
/cc @ampproject/wg-caching

Reply:
Rule change LGTM. Two nits re: the comment:
 1. In addition to "experimental", link to an open bug, so future maintainers can know if it's safe to delete the comment or the whole line (in case that task is left dangling).
 2. No need to say "email-only" as it's specified in the html_format and spec_name above.

Also, are you planning on updating user-facing documentation for this? If so, people may start depending on it, in which case it'll no longer be experimental. To avoid that, consider putting this behind an origin trial or experiment flag.

Reply:
Thanks Devin. Not planning to update user documentation for this (effectively a dark launch).